README.md

```diff
@@ -18,21 +18,52 @@
 
 ## :rocket: Getting started
 
-### Install Iter8 CLI
+### Using Iter8 Action
+
+Configure experiment with Helm values file and call:
+
+``` yaml
+    - run: |
+        cat << EOF > expConfig.yaml
+          url: https://httpbin.org/get
+          SLOs:
+            http/error-rate: 0
+            http/mean-latency: 100
+        EOF
+    - uses: iter8-tools/iter8@v0.9
```

Comment:
Is this simply a docker file ref? Or a git ref?

Could you help understand the components of this token `iter8-tools/iter8@v0.9`? How is `v0.9` used to get the "right version" of Iter8?

What are the steps involved when we release v0.10.0 of Iter8  CLI and want to update the GH action also to v0.10?


***

Reply:
It is meant here as a github ref. The formati is `{owner}/{repo}@{ref}` It can be many things. See https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsuses
I expect that if e release v0.10.0 we would the action reference to v0.10. We would also update action/Dockerfile to use iter8:v0.10

Reply:
This will not work.
***
If we release `v0.10.0`, then `v0.10` does not automatically become a valid git ref. Only `v0.10.0` is a valid git ref.
***
This means, `iter8-tools/iter8@v0.10` is an invalid action, but `iter8-tools/iter8@v0.10.0` is the valid action.

Reply:
I  saw https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsuses.

It says, major version is ok. But it doesn't say minor version is ok. It does say specific version is ok.